app/clients/sms/twilio.py

```diff
@@ -100,6 +104,8 @@ def send_sms(self, to, content, reference, **kwargs):
 
                 self.logger.info(f"Twilio message created using messaging_service_sid")
 
+                return message.sid
```

Comment:
If I understand correctly, send_sms return Optional[str].  I can only know that by reading through the code.  I would add an annotation to the function signature and discuss this in the function's doc string.

Reply:
Yep. Good point. I'll add it to the doc string.